Polygon.js

```diff
@@ -0,0 +1,167 @@
+{
+	"translatorID": "fa7c37b1-fda4-418a-a8b8-2491929411ab",
+	"label": "Polygon",
+	"creator": "czar",
+	"target": "^https?:\\/\\/(www.)?polygon.com",
```

Comment:
You've lost a bunch of things that were right before. You want
`^https?://(www\.)?polygon\.com`
in scaffold, which will then give you 
`^https?://(www\\.)?polygon\\.com`
in the actual translator.